xml/admin_monitoring.xml

```diff
@@ -415,6 +415,10 @@ Adding password for user admin
       </listitem>
      </itemizedlist>
 <screen language="yaml">
+# Enable the creation of pod security policy
+podSecurityPolicy:
+  enabled: true
```

Comment:
I think better would be to change it in our Prometheus Helm charts 
https://github.com/SUSE/kubernetes-charts-suse-com/blob/master/stable/prometheus/values.yaml#L5

Reply:
I will take care of it

Reply:
This change is for v3 where we use the upstream prometheus chart. Is prometheus available from the suse registry for v3?

Reply:
ahhh, I am sorry I didn't notice that it comes from v3. Yes, we are shipping now our own Helm charts and prometheus container in registry. Just wondering if we update documentation for v3 too.

Reply:
I have submitted PR to SUSE Prometheus helm charts https://github.com/SUSE/kubernetes-charts-suse-com/pull/105